shared/js/background/experiments.es6.js

```diff
@@ -34,17 +34,14 @@ class Experiment {
             .then(this.getVariant.bind(this))
             .then(this.getATBVariant.bind(this))
             .then(() => {
-                // Only set active experiment once
-                if (settings.getSetting('activeExperiment')) return
```

Comment:
I removed this so that we can get users out of an experiment group by removing the definition from shared/data/experiments-out.js